web-client/public/js/graph.js

```diff
@@ -251,13 +240,11 @@ var drawGraph = function (nodes, links, positiveWeights, negativeWeights, sheetT
 
   $container.on("mousedown", function () {
      if (scrolling) {
-       if ($container.hasClass("cursorGrab")) $container.removeClass("cursorGrab");
-       if (!$container.hasClass("cursorGrabbing")) $container.addClass("cursorGrabbing");
+       $container.removeClass(CURSOR_CLASSES).addClass("cursorGrabbing");
      }
   }).on("mouseup", function () {
     if (scrolling) {
-      if ($container.hasClass("cursorGrabbing")) $container.removeClass("cursorGrabbing");
-      if (!$container.hasClass("cursorGrab")) $container.addClass("cursorGrab");
+      $container.removeClass(CURSOR_CLASSES).addClass("cursorGrab");
```

Comment:
_And_ you remembered to chain—this is as concise as it gets, while remaining readable.